src/util/command_line_handling.cpp

```diff
@@ -679,7 +679,7 @@ namespace hpx { namespace util
         affinity_domain_ = detail::handle_affinity(cfgmap, vm, "pu");
         ini_config += "hpx.affinity=" + affinity_domain_;
 
-        affinity_bind_ = detail::handle_affinity_bind(cfgmap, vm, "");
+        affinity_bind_ = detail::handle_affinity_bind(cfgmap, vm, "balanced");
         if (!affinity_bind_.empty())
```

Comment:
On 05/25/2016 11:41 AM, John Biddiscombe wrote:
> In src/util/command_line_handling.cpp
> <https://github.com/STEllAR-GROUP/hpx/pull/2180#discussion_r64543813>:
>
>> @@ -679,7 +679,7 @@ namespace hpx { namespace util
>>          affinity_domain_ = detail::handle_affinity(cfgmap, vm, "pu");
>>          ini_config += "hpx.affinity=" + affinity_domain_;
>>
>> -        affinity_bind_ = detail::handle_affinity_bind(cfgmap, vm, "");
>> +        affinity_bind_ = detail::handle_affinity_bind(cfgmap, vm, "balanced");
>>          if (!affinity_bind_.empty())
>
> Does the empty check still have a purpose since you've set a balanced
> default?

Good catch!

>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/STEllAR-GROUP/hpx/pull/2180/files/ddd1295164834781cdedaf198b85db7b098a7a95#r64543813>
>